src/bsoncxx/string/view_or_value.hpp

```diff
@@ -14,13 +14,13 @@
 
 #pragma once
 
-#include <bsoncxx/config/prelude.hpp>
-
 #include <string>
 
 #include <bsoncxx/stdx/string_view.hpp>
 #include <bsoncxx/view_or_value.hpp>
 
+#include <bsoncxx/config/prelude.hpp>
```

Comment:
This is a publicly visible header, so it should get the public prelude header. .cpp files and private headers get the private prelude.